spec/mailers/user_mailer_spec.rb

```diff
@@ -640,7 +640,7 @@
     let(:email) { UserMailer.abuse_report(report.id) }
 
     it "has the correct subject" do
-      expect(email).to have_subject "[#{ArchiveConfig.APP_SHORT_NAME}] Your abuse report"
+      expect(email).to have_subject "[#{ArchiveConfig.APP_SHORT_NAME}] Abuse - #{report.summary}"
```

Comment:
Since the emails are translated now, there should been a test for missing translations added as well. We usually do this by putting `it_behaves_like "a translated email"` after `it_behaves_like "a multipart email"`.